### PR TITLE
vagrantfile fixes - US109034

### PR DESCRIPTION
**Pull request by RichWellum**
- Cleaned up embedded vagrantfile (mainly comments)
- Fixed a bug in setup
- Clean up Vagrantfile in test module.

This was also a research into:
- Question regarding why the need to loop when disconnecting serial? 
  config.vm.provider 'virtualbox' do |v|
- Potentially move id: ‘ssh’ => ‘XR Console’ - but where is the string visible?

I can confirm that the 'id' has to be ssh - I tried 'xrv64' 'XR Console' etc and it wouldn't load.

Similarly the serials needs to be embedded in a 'config.vm.provider 'virtualbox' do |v|' block.

### CODE REVIEW

File: include/embedded_vagrantfile [hunk 1]
```diff
@@ -9,22 +9,14 @@
...
   config.vm.synced_folder '.', '/vagrant', disabled: true
 
   # Give IOS XRv (64-bit) 400 seconds to come up
   config.vm.boot_timeout = 400
 
-  # Internal network for GE interfaces.
-  # config.vm.network "private_network", type: "dhcp"
-
-  # 57722 is the operns_sshd (22 is XR)
+  # Port 57722 is XR linux (operns_sshd)
```
Comment (glennmatthews) [inline include/embedded_vagrantfile]: `sshd_operns`

File: include/embedded_vagrantfile [hunk 2]
```diff
@@ -9,22 +9,14 @@
...
   config.vm.network :forwarded_port, guest: 57722, host: 2222, auto_correct: true
+  # Port 2 is XR Console
```
Comment (glennmatthews) [inline include/embedded_vagrantfile]: `22`

**Comment by glennmatthews** [comment]
:shipit:

File: include/embedded_vagrantfile (hunk 1 above)
Comment (RichWellum) [inline include/embedded_vagrantfile]: Ack.

File: include/embedded_vagrantfile (hunk 2 above)
Comment (RichWellum) [inline include/embedded_vagrantfile]: Ack

**Pull request merged**